release/models/isis/openconfig-isis-lsp.yang

```diff
@@ -34,7 +34,13 @@ submodule openconfig-isis-lsp {
     Section 4.c of the IETF Trust's Legal Provisions Relating
     to IETF Documents (http://trustee.ietf.org/license-info).";
 
-  oc-ext:openconfig-version "1.6.2";
+  oc-ext:openconfig-version "1.6.3";
```

Comment:
```suggestion
  oc-ext:openconfig-version "1.7.0";
```

Patch revision is only for minor corrections with no change or addition to intentions.   Since this add news leaves, and is non-breaking, the 'minor' version should be incremented.